content/en/blog/2023/how_to_debug_large_db_size_issue.md

```diff
@@ -0,0 +1,233 @@
+---
+title: How to debug large db size issue?
+author:  "[Benjamin Wang](https://github.com/ahrtr), VMware"
+date: 2023-01-04
+draft: false
+---
+
+# Background
+Users can configure the quota of the backend db size using flag `--quota-backend-bytes`. It's the max number of bytes 
+the etcd db file may consume, namely the ${etcd-data-dir}/member/snap/db file. Its default value is 2GB, and the 
+suggested max value is 8GB.
+
+2GB is usually sufficient for most use cases. If you run out of the db quota, it would be better to figure out 
+whether it's expected. It's exactly the reason why I provide this guide.
```

Comment:
It would be good to provide the `database space exceeded ...` error message as part of the issue.

Reply:
Added error message and also `NOSPACE` alarm example. thx